### DISCUSSION TITLE
ikea styrbar blueprint and MA players

### DISCUSSION THREAD

**Discussion by userMak** [music-assistant/support#3151] [Q&A] [answered]
Hi
I have an ikea styrbar button and I have configure it with this blueprint. https://community.home-assistant.io/t/ikea-styrbar-e2001-2002-ultimate-zigbee2mqtt-z2m/625090

Now I need to with a button click to start music assistant and play a playlist. 
The problem is that in HA the player is named as media_player.chromecast_audio and in music assistant as media_player.bose

![image](https://github.com/user-attachments/assets/114520b1-b3ee-43d6-89e6-4b462e5952fb)

in the blueprint though I can not select the media_player.bose (it doesn't appear) I can select only the media_player.chromecast_audio

 I don't think I can manual edit the blueprint and to be honest since I would like to easily edit the buttons through UI I would like to keep it as it is now.

Is there anything I could do about it?

**edit: OK. I made a script.. thanks**

**Comment by OzGav** [comment, marked as answer]
You should be able to just change the entity names as well

**Reply by userMak** [reply]
How I could that ? in the below menu?
![image](https://github.com/user-attachments/assets/861ef76c-f7bd-43e4-8e36-7df51e4351b1)

and how should I name the entity? chromecast_audio?

**Reply by OzGav** [reply]
You do that via the HA settings . Edit: and yes change the original chromecast_audio entity to something else and then change the one you need to chromecast_audio
